packages/types/package.json

```diff
@@ -74,7 +74,8 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@chainsafe/ssz": "^0.14.0",
-    "@lodestar/params": "^1.14.0"
+    "@lodestar/params": "^1.14.0",
+    "@lodestar/utils": "^1.14.0"
```

Comment:
Alternative would be to just add ethereum-cryptography here and move checksum encoding function here as well.

Not sure if it worth it as `@lodestar/utils` is quite light weight, maybe someone else a has stronger opinion on this